sentinel/src/schedule/outbound.js

```diff
@@ -0,0 +1,253 @@
+const objectPath = require('object-path'),
+      urlJoin = require('url-join'),
+      request = require('request-promise-native'),
+      vm = require('vm');
+
+const configService = require('../config'),
+      db = require('../db'),
+      logger = require('../lib/logger'),
+      lineage = require('@medic/lineage')(Promise, db.medic);
+
+const CONFIGURED_PUSHES = 'outbound';
+
+const fetchPassword = key =>
+  request(`${db.serverUrl}/_node/${process.env.COUCH_NODE_NAME}/_config/medic-credentials/${key}`)
```

Comment:
I think this should be `environment.serverUrl` as `db.serverUrl` no longer exists. This is likely due to a bad merge. @kennsippell What do you think?

Reply:
Can you help me understand a bit better? I see `db.serverUrl` defined [here](https://github.com/medic/medic/blob/master/sentinel/src/db.js#L50).

Reply:
Sorry, you're absolutely right. I misread `sentinel` as `api` for some reason... Sorry!

Reply:
No worries!! Thanks for checking this out.